docs/js/copyright.js

```
@@ -0,0 +1,8 @@
+// This file is generated at build time
+var copyrightText = "© Copyright 2025. Last updated on Mar 06, 2025.";
```

Biome (1.9.4)

[error] Use let or const instead of var.

A variable declared with var is accessible in the whole module. Thus, the variable can be accessed before its initialization and outside the block where it is declared.
See MDN web docs for more details.
Unsafe fix: Use 'const' instead.


(lint/style/noVar)